crates/database/src/auction_orders.rs

```diff
@@ -0,0 +1,54 @@
+use {
+    crate::{auction::AuctionId, OrderUid},
+    sqlx::PgConnection,
+};
+
+pub async fn insert(
+    ex: &mut PgConnection,
+    auction_id: AuctionId,
+    orders: &[OrderUid],
+) -> Result<(), sqlx::Error> {
+    const QUERY: &str = r#"INSERT INTO auction_orders (auction_id, order_uids) VALUES ($1, $2);"#;
+    sqlx::query(QUERY)
+        .bind(auction_id)
+        .bind(orders)
+        .execute(ex)
+        .await?;
+    Ok(())
+}
+
+pub async fn fetch(
+    ex: &mut PgConnection,
+    auction_id: AuctionId,
+) -> Result<Option<Vec<OrderUid>>, sqlx::Error> {
```

Comment:
This type looks a bit quirky (we should at least document what the difference between `Err(_)`, `Ok(None)` and `Ok(Some(vec[]))` is.

I think we could use `fetch_one` and treat `sqlx::Error::RowNotFound` in the persistence layer if we want to simplify this return type, but no strong feelings.